array.c

```diff
@@ -7411,20 +7378,14 @@ rb_ary_all_p(int argc, VALUE *argv, VALUE ary)
  *    [0, 1, 2].none? {|element| element > 3 } # => true
  *    [0, 1, 2].none? {|element| element > 1 } # => false
  *
- *  Does not call the block if +self+ is empty:
- *    [].none? {|element| fail 'Cannot happen' } # => true
- *
  *  If argument +obj+ is given, returns +true+ if <tt>obj.===</tt> no element, +false+ otherwise:
  *    ['food', 'drink'].none?(/bar/) # => true
  *    ['food', 'drink'].none?(/foo/) # => false
  *    [].none?(/foo/) # => true
  *    [0, 1, 2].none?(3) # => true
  *    [0, 1, 2].none?(1) # => false
  *
- *  Issues a warning ('given block not used') if both an argument and a block given:
- *    [0, 1, 2].none?(/foo/) { |element| } # => true
- *
- *  See also Enumerable#none?
+ *  Related: Enumerable#none?
```

Comment:
Should this be `Related: Enumerable#one?` or something else?

Reply:
Fixed.